catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -273,6 +326,32 @@ private void parseValidators(Map<String, Object> root, Outer outer) {
         return staged;
     }
 
+    private List<Callable<Boolean>> parseMetacardValidators(Changeset changeset,
```

Comment:
nit: the method is also called "parseMetacardValidators" but it seems like this is really registering the validators.  Maybe consider changing this method name.